Source/CMakeLists.txt

```diff
@@ -215,7 +215,7 @@ endif()
 install(FILES ${WOOF_DLLS} DESTINATION .)
 
 # Try get dependencies in MSYS2 environment.
-if(MSYS)
+if(WIN32 AND NOT SDL2_DIR)
```

Comment:
I think it doesn't work anywhere except MSYS2.

Reply:
Well, this logic works on Linux, on Linux when cross-building for Windows and in my native Windows shell using MingW.

Reply:
> native Windows shell using MingW.

native MSYS2 shell using MinGW